internal/testrunner/runners/system/runner.go

```diff
@@ -438,7 +438,9 @@ func (r *runner) runTest(config *testConfig, ctxt servicedeployer.ServiceContext
 
 	// Validate fields in docs
 	fieldsValidator, err := fields.CreateValidatorForDataStream(serviceOptions.DataStreamRootPath,
-		fields.WithNumericKeywordFields(config.NumericKeywordFields))
+		fields.WithNumericKeywordFields(config.NumericKeywordFields),
+		fields.WithDisabledAllowedIPCheck(),
```

Comment:
nit: I would leave a comment here explaining why is it disabled for system tests.